docs/products/postgresql/howto/pagila.rst

```diff
@@ -0,0 +1,258 @@
+Sample dataset: Pagila
+======================
+
+Aiven provides a sample database you can import in your Aiven for PostgreSQL service. This page covers information about the database and the procedure to get it up and running.
+
+Pagila is a PostgreSQL port of the `Sakila Sample Database <https://dev.mysql.com/doc/sakila/en/>`_. The examples here use one from ``devrimgunduz``, `version 2.1.0 <https://github.com/devrimgunduz/pagila>`_.
+
+Sakila (and thus, Pagila) is a database representing a DVD rental store (remember those?!), containing information about films (like title, category, actresses), rental stores (like address, staff members, customers) and rentals, where a customer rents a film from a store through its staff.
+
+With all these relational information, Pagila is a perfect fit to play around with PostgreSQL and the SQL language.
+
+Load Pagila on PostgreSQL
+-------------------------
+
+Before exploring the Pagila database, follow the :doc:`create new service article<../../../platform/howto/create_new_service>` to spin up a PostgreSQL instance.
+
+With your new PostgreSQL service, you can quickly load the sample dataset by clicking on the "Load sample dataset" button.
+
+.. image:: /images/products/postgresql/pagila-load-sample-dataset.png
+    :alt: A screenshot of the Aiven Console showing the "Load sample dataset" button.
+
+After loading the data, connect to the PostgreSQL instance using the command below. The ``SERVICE_URI`` value can be found in the Aiven Console dashboard.
+
+.. code:: shell
+
+   psql 'SERVICE_URI'
+
+You can also load it manually by following the steps below:
+
+1. Download the ``pagila-data.sql`` from our `GitHub repository <https://github.com/aiven/devportal/blob/paglia-sample-database/code/products/postgresql/pagila/pagila-data.sql>`_.
+
+.. Tip::
+    You may use the following command on your terminal::
+
+        wget https://raw.githubusercontent.com/aiven/devportal/paglia-sample-database/code/products/postgresql/pagila/pagila-data.sql
+
+2. Connect to the PostgreSQL instance using the command below. The ``SERVICE_URI`` value can be found in the Aiven Console dashboard.
+
+.. code:: shell
+
+   psql 'SERVICE_URI'
+
+3. Within the ``psql`` shell, create a database named ``pagila`` and connect to it with the command below:
+
+.. code:: psql
+
+    CREATE DATABASE pagila;
+    \c pagila;
+
+4. Populate the database with the command below. This might take some time.
+
+.. code:: psql
+
+    \i pagila-data.sql;
+
+5. Once the command finishes, make sure to reconnect to the database to access the imported data:
+
+.. code:: psql
+
+    \c pagila;
+
+**You are ready to go!** You can use the :ref:`Sample queries<sample_queries>` section below to explore the database. Have fun!
+
+Entity-relationship model diagram
+---------------------------------
+
+The image below shows an overview of the Pagila database tables and views, generated by `DBeaver <https://dbeaver.io>`_.
```

Comment:
Should we add a bit more wording here? The picture is nice, but an unexperienced user might struggle to find the information needed. 
Something like "the actors table contains ...., and can be joined with the film table via the `film_actor` table", we don't need to be exaustive, but just giving the end reader a bit more knowledge on why we're building the queries like we do.

I often see the struggle in stackoverflow being in understanding the datamodel, before even starting with sql

Reply:
Done!